### PR TITLE
ipld: move for dag service to block service

### PR DESCRIPTION
**Pull request by vgonkivs**

### CODE REVIEW

File: ipld/retriever.go
Comment (Wondertan) [inline ipld/retriever.go:99]: We should add additional NodeAdder constructor from Blockservice, context and size

File: ipld/plugin/nmt.go
Comment (Wondertan) [inline ipld/plugin/nmt.go:187]: GetNode s.t. `plugin.GetNode`

**Review by liamsi — APPROVED** [review]
I think @Wondertan should have a final say here but this LGTM.

**Review by Wondertan — APPROVED** [review]
Very nice. Thank you. 
Just a few prettifying nits.

Also, this PR does not fully solve the attached issue. It is just a first step.

File: ipld/plugin/nmt.go
Comment (vgonkivs) [reply ipld/plugin/nmt.go]: done

File: ipld/retriever.go
Comment (vgonkivs) [reply ipld/retriever.go]: Changed `NewNmtNodeAdder` ctor.

File: node/p2p/ipld.go
Comment (renaynay) [inline node/p2p/ipld.go:9]: Is it IPFS?

**Review by renaynay — APPROVED** [review]
lgtm with one comment

File: node/p2p/ipld.go
Comment (vgonkivs) [reply node/p2p/ipld.go:9]: `github.com/ipfs/go-blockservice`

**Pull request merged**
